src/neural/CL/OpenCLTuner.cc

```diff
@@ -30,7 +30,7 @@
 #include "OpenCLParams.h"
 #include "OpenCLTuner.h"
```

Comment:
Full path, or square brackets, depending on what is meant.

Reply:
Ok  - I do it now. But it's now pretty sure we will have a conflict between the 2 PR.

Reply:
Hm, I don't see any changes, did you forget to push your commit?

Reply:
Found them in another PR!